jqqb/rule.py

```diff
@@ -15,49 +15,55 @@ def __init__(self, rule_dict):
         self.value = rule_dict["value"]
 
     def evaluate(self, obj):
-        result = self.get_operator()(self.get_input(obj), self.get_value())
+        result = self.get_operator()(
+            self.get_input(obj), self.get_value_custom_fields(obj)
+        )
         return result
 
     def inputs(self, obj):
         inputs = self.get_input(obj)
         return inputs
 
-    def values(self):
-        values = self.get_value()
+    def values(self, obj={}):
```

Comment:
Be careful using mutables in argument default values: https://www.codecademy.com/learn/learn-intermediate-python-3/modules/int-python-function-arguments/cheatsheet